### ISSUE TITLE
Mass Mailing not showing value open or clicks or receive the report

### ISSUE THREAD

**Issue by prakashjain** [odoo/odoo#24640]
Impacted versions:
Odoo 11
Steps to reproduce:
Mass Mail Module Installed 
Created Mass Mail from Pipeline list view  using wizard Lead/Importunity Mass Mailing 

Current behavior:
Mass Mailing Delivered updated, but Opened, Clicks and Replied not updated always shows 0%

Expected behavior:
Mass Mailing Delivered Opened, Clicks and Replied also need to update based on user actions.

Video/Screenshot link (optional):

**Comment by prakashjain** [comment]
Mass Mailing Opened, Clicked and Replied checked in version 10 it always shows 0%, Is any missing configuration or this functionality not available in Odoo?  Mail delivered and send count shows correctly.

**Comment by Yenthe666** [comment]
Hi @prakashjain,

Thanks for your report.
Do you have multiple databases? Tracking of e-mail statistics only works with one database or a specific `db-filter` to handle this.

Regards,
Yenthe

**Comment by prakashjain** [comment]
Yes i have multiple databases. --db-filter applied but still open and clicks email statistics not working.

**Comment by prakashjain** [comment]
Hello Yenthe,
While debugging Mass Mail controllers "track_mail_open"  method not called.
Create method created url http://public-IP-Id/track.mail  and param database name. when manually open url in browser "Opened" count increased. 
"track_mail_open"   how to make auto call at the time of user click mail ?
 The above method will call if odoo run with localhost or static public ip address? or need to need run with domain name?  
And also noticed odoo send email (mail.mail) record auto deleted it will create any problem?
Thanks

**Comment by Yenthe666** [comment]
@prakashjain it should work fine out of the box. It should point to the URL where your Odoo is publicly available on, usually this is the domain.
I'm not sure what your question is about the mail.mail records though.

**Comment by prakashjain** [comment]
Thanks for reply.

We are using a Public IP address, odoo publicly available on (in that case opened or replied email statistics will not working, please confirm only it will work domain not public ip address?).

I have other questions related this topic Email going to Spam.
We tried two different Outgoing Server 
1) "marketing@companydomain.com" Outgoing configuration email going to the Spam folder. 
2) "username@companydomain.com" Outgoing configuration email going to the Inbox.
Same domain marketing email ID going to spam, please let me know why this happening?

**Comment by Yenthe666** [comment]
@prakashjain as long as you point to the correct public IP it should work fine too.

Regarding the e-mail severs: this depends on the mail provider, configurations, relaying, the content and so much more.. I don't have a one liner answer or solution for you there.

**Comment by prakashjain** [comment]
Hi Yenthe,

No of "Opened" email not working we configured correct public IP. 
 Email Content --> Inspect Element --> After Copy and Paste Url Manually "Opened" mail working.
In Gmail settings (outgoing mail server ID)  in any options to Mail tracking ON or odoo settings tracking On options available?

**Comment by stidiovip** [comment]
After installation, the module does not display.
